open-api/rest-catalog-open-api.yaml

```diff
@@ -1581,6 +1607,17 @@ components:
         type: string
       example: [ "accounting", "tax" ]
 
+    PageToken:
+      description:
+        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables) by signaling to the service
+        that they would prefer the response to be paginated.
+
+        Clients will initiate the request by sending an empty pageToken e.g. GET /tables?pageToken or /tables?pageToken=
+        For servers that support pagination, they will recognize pageToken and return a next pageToken in response if there are more results available.
```

Comment:
```suggestion
        Servers that support pagination will recognize page-token and return a next-page-token in the response if there are more results available.
```

Reply:
I believe it should be `pageToken` and not `page-token`.